hpat/hiframes/hiframes_typed.py

```diff
@@ -866,7 +866,7 @@ def _run_call_series(self, assign, lhs, rhs, series_var, func_name):
             return self._replace_func(func, [data], pre_nodes=nodes)
 
         if func_name in ('std', 'nunique', 'describe', 'isna',
-                         'isnull', 'median', 'idxmin', 'idxmax', 'unique'):
+                         'isnull', 'idxmin', 'idxmax', 'unique'):
```

Comment:
usually I do following https://github.com/IntelPython/hpat/pull/189/files#diff-f13c21aac60fb8f7ae3c9527239d1e17R990
instead removing this name from this loop

Reply:
`_not_series_array_attrs = ['resolve_median', ...]` just avoids replacement of `series.median` with `array.median`. As I see we are not able to keep both approaches just by adding `'resolve_median'` to the list. Please correct me if I'm wrong.

Reply:
@shssf @densmirn No, if we keep 'median' here we'll use and invoke the old-style implementation (so commenting hpat.hiframes.api.median in series_kernels would cause lowering error). It's not possible to keep both implementations (unless one or the other is selected basing on configured pipeline). But anyway, in the new-style the parallel test is broken, even if I call the old-style hpat.hiframes.api.median function from the overload itself.